src/common/components/forms/CancelOfferForm.tsx

```diff
@@ -48,10 +52,31 @@ const CancelOfferForm: FC<CancelOfferFormProps> = ({
     if (!publicKey || !signTransaction || !offer || !nft) {
       return;
     }
+
+    const newActions: Action[] = [
+      {
+        name: `Canceling offer for ${Number(offer.price) / LAMPORTS_PER_SOL} SOL...`,
+        id: `cancelOffer`,
+        action: onCancelOffer,
+        param: undefined,
+      },
+    ];
+
     try {
-      await onCancelOffer();
-      toast.success(`Confirmed cancel offer success`);
-      await refetch();
+      await runActions(newActions, {
+        onActionSuccess: async () => {
+          toast.success(`Confirmed cancel offer success`);
```

Comment:
Is the order important? In other components you show success _after_ `await refetch();`

Reply:
order shouldn't matter